### PR TITLE
Dev nav bar

### PR DESCRIPTION
**Pull request by SebastienFrancois**
Material UI Navbar plus responsive Drawer feature. Working on react router!

### CODE REVIEW

**Review by davidbourrel — APPROVED** [review]

File: src/components/Navigation/Navigation.jsx [hunk 1]
```diff
@@ -1,28 +1,207 @@
...
+import React, { useState } from 'react';
+import {
+  AppBar,
+  Typography,
+  Tabs,
+  Tab,
+  Toolbar,
+  // Button,
+  // Menu,
+  // MenuItem,
```
Comment (Froyj) [inline src/components/Navigation/Navigation.jsx:15]: ejecter le code mort

File: src/components/Navigation/Navigation.jsx [hunk 2]
```diff
@@ -1,28 +1,207 @@
...
-      </li>
-      <li>
-        <Link to="/perseverance">Perseverance</Link>
-      </li>
-    </ul>
-  </nav>
-);
+const Navigation = () => {
+  const [value, setValue] = useState(0);
+  // const [anchorEl, setAnchorEl] = useState(false);
```
Comment (Froyj) [inline src/components/Navigation/Navigation.jsx:23]: code mort =  delete

File: src/components/Navigation/Navigation.jsx [hunk 3]
```diff
@@ -1,28 +1,207 @@
...
+
+  const handleClickTab = (e, newValue) => {
+    setValue(newValue);
+  };
+  // const handleOpenMenu = (e) => {
+  //   setAnchorEl(e.currentTarget);
+  // };
+  // const handleCloseMenu = (e) => {
+  //   setAnchorEl(false);
+  // };
```
Comment (Froyj) [inline src/components/Navigation/Navigation.jsx:33]: code mort =  delete

File: src/components/Navigation/Navigation.jsx [hunk 4]
```diff
@@ -1,28 +1,207 @@
...
+            className: "tab-icons"
+          }}
+        >
+          <FaRobot />
+        </IconContext.Provider>
+      }
+      label="Spirit"
+    />
+  </MenuItem>
+    </Menu> */
```
Comment (Froyj) [inline src/components/Navigation/Navigation.jsx:207]: code mort =  delete

File: src/components/Navigation/components/Drawermenu/DrawerMenu.jsx [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+          </ListItem>
+
+          {/* <Button
+            area-controls="menu"
+            variant="contained"
+            color="secondary"
+            size="large"
+          >
+            Explore
+          </Button> */}
```
Comment (Froyj) [inline src/components/Navigation/components/Drawermenu/DrawerMenu.jsx:114]: code mort =  delete

File: src/components/Navigation/Navigation.jsx (hunk 1 above)
Comment (SebastienFrancois) [reply src/components/Navigation/Navigation.jsx]: Done

File: src/components/Navigation/Navigation.jsx (hunk 2 above)
Comment (SebastienFrancois) [reply src/components/Navigation/Navigation.jsx]: Done

File: src/components/Navigation/Navigation.jsx (hunk 3 above)
Comment (SebastienFrancois) [reply src/components/Navigation/Navigation.jsx]: Done

File: src/components/Navigation/Navigation.jsx (hunk 4 above)
Comment (SebastienFrancois) [reply src/components/Navigation/Navigation.jsx]: Done

File: src/components/Navigation/components/Drawermenu/DrawerMenu.jsx (hunk 5 above)
Comment (SebastienFrancois) [reply src/components/Navigation/components/Drawermenu/DrawerMenu.jsx:114]: Done

**Review by Froyj — APPROVED** [review]

**Pull request merged**